mobile/lib/components/ra_bottomnavbar.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:flutter/material.dart';
+import 'package:radioaktywne/extensions/build_context.dart';
+
+import 'package:radioaktywne/resources/assets.gen.dart';
+import 'package:radioaktywne/resources/colors.dart';
+
+class RaBottomNavigationBar extends StatelessWidget {
```

Comment:
adjustment for using useState (and other hooks in general)
```suggestion
class RaBottomNavigationBar extends HookWidget{
```